### PR TITLE
PD-2474: Changed warptools docker for all Metrics tasks

### PR DESCRIPTION
**Pull request by aawdeh**
This is to incorporate the changes in the warp-tools repo -- which removes reads on MT genes from the Multiome intronic, exonic, and intergenic counts. 

Changes in warp-tools repo can be found here: https://github.com/broadinstitute/warp-tools/pull/100

-  Added gtf as input parameter for CalculateGeneMetrics and CalculateUMIMetrics tasks.
- Updated warptools docker in the all the metric, h5ad and fastqprocess tasks.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]

File: tasks/skylab/Metrics.wdl
Comment (nikellepetrillo) [inline tasks/skylab/Metrics.wdl:11]: do you mind rebuilding the docker using the `warp-tools:1.0.9-blahblahblah` nomenclature and using that here instead of `aa-mt-tagsort`?

Comment (aawdeh) [reply tasks/skylab/Metrics.wdl:11]: Yes of course. Im going to rebuild the docker with an actual version number once the warp-tools PR gets merged.

**Comment by aawdeh** [comment]
Smart tests are expected to fail because we are not accounting for MT genes. CompareGeneMetrics, CompareCellMetrics and CompareH5adFilesOptimus are failing.

**Comment by nikellepetrillo** [comment]
looks good, but i noticed one of the optimus plumbing samples was failing because of an out of disk error:  https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/beb40f6f-52f6-4b61-b81a-b30b63d3985f/metadata
I found this error in the GeneMetrics log:
```
/cromwell_root/script: line 59: echo: write error: No space left on device
```
do we need to add more disk to this task?

**Review by kayleemathews — COMMENTED** [review]
Docs are updated!

**Comment by aawdeh** [comment]
> looks good, but i noticed one of the optimus plumbing samples was failing because of an out of disk error: https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/beb40f6f-52f6-4b61-b81a-b30b63d3985f/metadata I found this error in the GeneMetrics log:
> 
> ```
> /cromwell_root/script: line 59: echo: write error: No space left on device
> ```
> 
> (link to cromwell run: https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/beb40f6f-52f6-4b61-b81a-b30b63d3985f/metadata) do we need to add more disk to this task?

That looks like the error I was getting when I was printing everything. That is no longer an issue now tho as I removed the print statements.

**Comment by nikellepetrillo** [comment]
looks good, just need to update changelogs/versions 👍 
```
18:09:17 Multiome.changelog.md has not been changed and needs to be updated
18:09:17 Optimus.changelog.md has not been changed and needs to be updated
18:09:17 PairedTag.changelog.md has not been changed and needs to be updated
18:09:17 SlideSeq.changelog.md has not been changed and needs to be updated

**Review by nikellepetrillo — CHANGES_REQUESTED** [review]
looks good, just need to update changelogs/versions and a change to the warp-tools:aa-mt-tagsort docker  👍 
```
18:09:17 Multiome.changelog.md has not been changed and needs to be updated
18:09:17 Optimus.changelog.md has not been changed and needs to be updated
18:09:17 PairedTag.changelog.md has not been changed and needs to be updated
18:09:17 SlideSeq.changelog.md has not been changed and needs to be updated

**Comment by aawdeh** [comment]
> looks good, just need to update changelogs/versions and a change to the warp-tools:aa-mt-tagsort docker 👍
> 
> ```
> 18:09:17 Multiome.changelog.md has not been changed and needs to be updated
> 18:09:17 Optimus.changelog.md has not been changed and needs to be updated
> 18:09:17 PairedTag.changelog.md has not been changed and needs to be updated
> 18:09:17 SlideSeq.changelog.md has not been changed and needs to be updated
> ```

Agreed, thanks! Just waiting to merge the warp-tools PR and merge the other warp PR so I don't have to change the changelogs/versions twice.

File: tasks/skylab/Metrics.wdl
Comment (nikellepetrillo) [inline tasks/skylab/Metrics.wdl:245]: should this be updated to us.gcr.io/broad-gotc-prod/warp-tools:2.0.1 ?

Comment (aawdeh) [reply tasks/skylab/Metrics.wdl]: Just updated it

**Review by nikellepetrillo — APPROVED** [review]

**Comment by aawdeh** [comment]
Holding off merging this PR until Megan's PRs are merged.

File: pipelines/skylab/multiome/atac.changelog.md
Comment (kayleemathews) [inline pipelines/skylab/multiome/atac.changelog.md:1]: @aawdeh would you mind updating the version number of the ATAC pipeline in the README before you merge? Thanks!

Comment (aawdeh) [reply pipelines/skylab/multiome/atac.changelog.md:1]: Done!

**Comment by aawdeh** [comment]
retest this please

**Comment by aawdeh** [comment]
retest this please

**Comment by aawdeh** [comment]
retest this please

**Comment by nikellepetrillo** [comment]
retest this please

File: pipelines/skylab/slideseq/SlideSeq.changelog.md
Comment (kayleemathews) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:1]: version number should probably by 3.0.1, right?

Comment (aawdeh) [reply pipelines/skylab/slideseq/SlideSeq.changelog.md:1]: I guess it depends on how we want to increase the version number. I think it was increased from 2.1.6 to 3.0.0 in the loom to h5ad PR. I just readjusted the version number for that change to 2.1.7 and then increased the one here to 2.1.8.

Comment (aawdeh) [reply pipelines/skylab/slideseq/SlideSeq.changelog.md]: I talked to the team about it, and we updated to 3.0.0 b/c a slideseq output was removed. I set it back to 3.0.0 and 3.0.1

**Pull request merged**
